addons/website/views/website_templates.xml

```diff
@@ -1301,7 +1301,7 @@
                                     <i class="fa fa-instagram m-1"/>
                                 </a>
                                 <a href="mailto:info@yourcompany.example.com" class="ms-3" target="_blank">
-                                    <i class="fa fa-1x fa-envelope"/>
+                                    <i class="fa fa-1x fa-envelope o_editable_media"/>
```

Comment:
I'm not sure where to find this `<i>` in the website but I think that `o_editable_media` should also be added here.

Reply:
You need to configure the website to change the header style. From the top of my head I think you need to click in the header then you should have options to configure the header in the sidebar. One of those header templates have the code that you see here. It might be called "magazine" because the template uses that name, but sometimes things are renamed from a user point of view but not in the data themselves so sometimes the names don't match, just a heads up.

I'm confused though. Why only change this particular one and not the others ?

Reply:
I only change the "custom" social media, others don't deserve the `o_editable_media` apparently (I wouldn't mind allowing users to change the icon for facebook/youtube etc but that's me).

Reply:
Then `o_editable_media` can't be used to distinguish the icons from custom social media links from the non-custom social media links. And we'll need an exception to the exception (the rule that says non-editable tags are editable if they have o_editable_media will need a further exception).